src/components/freelancer/interviewProfile/interviewProfile.tsx

```
@@ -475,37 +474,43 @@ const InterviewProfile: React.FC<{ freelancerId: string }> = ({
                       <TableCell className="">
-                        <Badge className={getBadgeColor(item.level)}>
+                        <Badge className={`px-3 py-1 rounded-full text-xs font-medium border ${statusOutlineClasses(item.level)}`}>
                           {item?.level?.toUpperCase()}
                         </Badge>
                       </TableCell>
-                      <TableCell className="">
-                        {typeof item.experience === 'number' &&
-                        item.experience > 0
-                          ? item.experience + ' years'
+                      <TableCell className="text-gray-600 dark:text-gray-300">
+                        {typeof item.experience === 'number' && item.experience > 0
```

ESLint

[error] Insert `⏎·······················`

(prettier/prettier)